for_other_cantons.py

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jan  3 09:20:09 2022
+
+@author: eduardoaraujo
+"""
+import pandas as pd 
+import numpy as np
+import streamlit as st
+import plotly.graph_objects as go
+from forecast import download_button
+from get_data import get_canton_data  
+from sqlalchemy import create_engine
+engine = create_engine("postgresql://epigraph:epigraph@localhost:5432/epigraphhub")
+
+
+
+#@st.cache
```

Comment:
@eduardocorrearaujo  We can turn on caching on some of these functions

Reply:
When I try to turn on this function I receive this message:

**AttributeError: 'QueuePool' object has no attribute '_pool'**

Reply:
This is due to an incompatibility between st.cache and Sqlalchemy. I have replaced it by cache tools

Reply:
Do you have an example of how to use these cache tools? Is It using the own streamlit st.chace function or other packages?